### PR TITLE
Add a rule to limit line length and ignore imports

### PR DESCRIPTION
**Pull request by mislavmiocevic**
Task: https://app.productive.io/1-infinum/task/2919330

### CODE REVIEW

File: packages/eslint-config-core-js/index.js [hunk 1]
```diff
@@ -21,5 +21,6 @@ module.exports = {
 		'no-underscore-dangle': ['error', { allowAfterThis: true }],
 		'no-void': 'error',
 		semi: 'error',
+		'max-len': ['error', { code: 140, ignorePattern: '^import .*' }],
```
Comment (mislavmiocevic) [inline packages/eslint-config-core-js/index.js:24]: 140 can be changed of course.

Comment (isBatak) [reply packages/eslint-config-core-js/index.js:24]: Why would we need to add this if we use prettier?

Comment (isBatak) [reply packages/eslint-config-core-js/index.js:24]: We have prettier rule in handbook https://infinum.com/handbook/frontend/react/getting-started/ecosystem#prettier and line lenght is fixated to 120. I think we should align linter with prettier.

Comment (mislavmiocevic) [reply packages/eslint-config-core-js/index.js:24]: It was a long time ago, but someone requested to have a linter ignore the warning and/or error for the long imports that prettier couldn't restructure.
I will put 120 then.

Comment (mislavmiocevic) [reply packages/eslint-config-core-js/index.js]: Done. It is now 120.

**Review by isBatak — APPROVED** [review]

**Review by adrianbicanic — APPROVED** [review]

**Pull request merged**